src/api_server.py

```diff
@@ -141,7 +80,11 @@ def read_root():
 def predict(request: PredictRequest):
     # get model prediction for the input request
     start_time = datetime.now()
-    predictions = classifier.predict(request.review_text)
+    predictions = (
+        classifier.predict(request.review_text)
+        if request.model_type == 1
+        else classifier.predict_seq2seq(request.review_text)
```

Comment:
Consider being explicit about the behaviour of the code when `model_type` is set to specific value. Limit the value that `model_type` can take allows us to extend the API for other model types later on.